tests/cupy_tests/sparse_tests/test_csc.py

```diff
@@ -467,8 +467,28 @@ class TestCsrMatrixGetitem(unittest.TestCase):
 
     @testing.numpy_cupy_equal(sp_name='sp')
     def test_getitem_int_int(self, xp, sp):
-        return _make(xp, sp, self.dtype)[0, 1]
+        self.assertEqual(_make(xp, sp, self.dtype)[0, 1], 1)
```

Comment:
Testing the return value with `numpy_cupy_equal` isn't enough? Anyway, numpy test is not necessary if you test with `assertEqual`.